scripts/bumpChangelogs/getNewChangelogContent.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (version, prLink, changesDescription) => existingChangelog => {
+  const changelogLines = existingChangelog.split('\n');
+  const [prNumber] = prLink.split('/').splice(-1);
+  const newChanges = `| ${version} | [PR#${prNumber}](${prLink}) ${changesDescription} |`;
+  const lineNumberToAddChanges =
+    changelogLines.indexOf('| Version | Description |') + 2;
```

Comment:
This feels flakey, anyone changing the spacing around `version` and `description` will break it, can it be less fussy? maybe just check those words exist?

Reply:
I used a regex here to match the table header and return exactly what that line is then used that to get the index. This will work with extra spaces but still has to be a markdown table row.